packages/rx_bloc_cli/CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 ## [4.0.0]
 Contains breaking changes
 * The `--enable-auth-matrix` option is removed in favour of the [--enable-mfa][mfa_doc_url] (Multi-Factor Authentication) capability.
-* The go_router package is upragded to version ^14.2.1
+* Updated `go_router` to `^14.2.1`
+* Updated `rx_bloc` to `^6.0.0`
+* Updated `rxdart` to `^0.28.0`
 
```

Comment:
Should we add flutter_rx_bloc and rx_bloc_generator?

Reply:
This was partially implemented due to the dependencies, so we'll have to change even more dependencies. But good catch! I'll list the most important ones.